src/FixedDataTable.js

```diff
@@ -773,14 +773,17 @@ var FixedDataTable = React.createClass({
     var columnBefore = this.state.columnReorderingData.columnBefore;
     var columnAfter = this.state.columnReorderingData.columnAfter;
     var reorderColumn = this.state.columnReorderingData.columnKey;
+    var cancelReorder = this.state.columnReorderingData.cancelReorder;
 
     this.setState({
       isColumnReordering: false,
       columnReorderingData: {}
     });
-    this.props.onColumnReorderEndCallback({
-      columnBefore, columnAfter, reorderColumn
-    });
+    if (!cancelReorder) {
+      this.props.onColumnReorderEndCallback({
+        columnBefore, columnAfter, reorderColumn
+      });
+    }
 
     var onHorizontalScroll = this.props.onHorizontalScroll;
```

Comment:
I don't think we want to run this onHorizontalScroll code here.
Can we just return early if cancelReorder is true?